src/app/docs/date-picker-docs/date-picker-docs.component.ts

```diff
@@ -13,6 +13,7 @@ import { ControlDocsAbstract } from '../control-docs.abstract';
   imports: [JsonFormsZorroModule, NzTableModule, NzCodeEditorModule, FormsModule, EditorFormatterPipe],
 })
 export class DatePickerDocsComponent extends ControlDocsAbstract {
+  override data = { date: '2024-02-22' };
```

Comment:
That will look weird to person who will use docs when date will be in past. I can simulate that when I change year to 2023

<img width="392" alt="Screenshot 2024-02-07 at 06 16 21" src="https://github.com/wojtek1150/json-forms-zorro-wrapper/assets/32877864/02805d78-c293-48b9-a70e-63f1b38cebce">